todo-app/src/App.js

```diff
@@ -1,13 +1,26 @@
-import React, { Component } from 'react';
+import React from 'react';
+import Tasks from './Tasks'
+import task from './TasksDB'
 
-class App extends Component {
+export default class App extends React.Component {
   render() {
     return (
       <div>
-        Hello World!
+        <h1>ToDo List:</h1>
+        <div>
+          {task.map((task, index) => {
```

Comment:
You could have imported this `task` as `tasks`, e.g. `import tasks from './TasksDB'`, so we avoid using the same variable name inside map and for this list.